packages/components/src/components/input-field/input-field.jsx

```diff
@@ -297,13 +297,9 @@ class InputField extends React.Component {
             <React.Fragment>
                 {!!prefix && (
                     <div className={classNamePrefix}>
-                        <span
-                            className={classNames(
-                                `${classNamePrefix}--symbol`,
-                                'symbols',
-                                `symbols--${prefix.toLowerCase()}`
```

Comment:
Is it ok to remove this last class?

Reply:
Actually we can remove all the `symbols` classes. We no longer use them.